R/internal.R

```diff
@@ -51,8 +51,27 @@ add_multiline_prefix <- function(prefix, as_is = FALSE) {
   )
 }
 
+
+
 enclose <- function(prefix, postfix = prefix) {
   a <- rstudioapi::getSourceEditorContext()
   for (s in a$selection)
     rstudioapi::insertText(location = s$range, text = sprintf("%s%s%s", prefix, s$text, postfix))
+  
+  return(a)
```

Comment:
can you return this as `invisible(a)` please. It shouldn't bother with your addin and will not have any unintended outputs in other functions.